packages/devtools-sham-modules/client/shared/key-shortcuts.js

```diff
@@ -2,12 +2,10 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-"use strict";
-
-
-const { Services } = require("devtools-modules");
+const { Services: { appinfo }} = require("devtools-modules");
 const EventEmitter = require("../../shared/event-emitter");
-const isOSX = Services.appinfo.OS === "Darwin";
+const isOSX = appinfo.OS === "Darwin";
```

Comment:
this also blows up in the unit tests because `appinfo` uses window.navigator under the hood